actions/dependabot-auto-triage/action.yml

```diff
@@ -0,0 +1,52 @@
+name: "Auto Dismiss Dependabot Alerts"
+description: "Auto dismiss Dependabot alerts based on manifest path"
+inputs:
+  token:
+    description: "GitHub token with permissions to dismiss alerts"
+    required: true
+  alert-types:
+    description: 'Comma-separated list of alert types to dismiss (default: "dependency")'
+    required: false
+    default: "dependency"
+  paths:
+    description: "Multi-line list of glob patterns to match manifest paths to dismiss"
+    required: true
+    # Example: |
+    #   terraform/modules/**/*.json
+    #   docker/vendor/**
+  dismissal-comment:
+    description: "Default comment to add when dismissing alerts"
+    required: false
+    default: "Auto-dismissed based on manifest path configuration"
+  dismissal-reason:
+    description: "Default reason for dismissal"
+    required: false
+    default: "not_used"
+    # Options: 'fix_started', 'inaccurate', 'no_bandwidth', 'not_used', 'tolerable_risk'
```

Comment:
Would it make sense to add the validation for this into the index.ts file?

Reply:
Done!